src/cpp/src/image_generation/flux_pipeline.hpp

```diff
@@ -358,28 +364,45 @@ class FluxPipeline : public DiffusionPipeline {
         float* timestep_data = timestep.data<float>();
 
         for (size_t inference_step = 0; inference_step < timesteps.size(); ++inference_step) {
```

Comment:
should not we measure time of whole iteration (inference of denoising model + scheduler overhead + some other steps like in inpainting) ?